docs/settings/reporting-settings.asciidoc

```diff
@@ -248,6 +248,11 @@ a| `xpack.reporting.capture.browser`
   exist. Configure this to a unique value, beginning with `.reporting-`, for every
   {kib} instance that has a unique <<kibana-index, `kibana.index`>> setting. Defaults to `.reporting`.
 
+| `xpack.reporting.capture.networkPolicy`
+  | Capturing a screenshot from a {kib} page involves sending out requests for all the linked web assets. For example, a Markdown
+  visualization can show an image from a remote server. You can configure what type of requests to allow or filter by setting a
+  Network Policy for Reporting. See <<reporting-network-policy, Network Policy>> for more details.
```

Comment:
missed this one the first time around.  Network Policy should be lower case.  How about removing the last sentence and making "network policy" the link in the previous sentence:

You can configure what type of requests to allow or filter by setting a <<reporting-network-policy, network policy>> for Reporting.  ~See <<reporting-network-policy, Network Policy>> for more details.~